### PR TITLE
✨Source Primetric: Migrate Python CDK to Low-code CDK

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
*Migrating Source Primetric to Low-Code CDK*
Closes airbytehq/airbyte-internal-issues#7029

## How
*Developed using (Configuration Based Source) low-code CDK*

## Recommended reading order
1. `spec.yaml`
2. `manifest.yaml`
3. `schemas/*`

## Tests

<details open><summary><strong>Integration & Acceptance</strong></summary>

 - Build Successfull
 
image

</details>

## 🚨 User Impact 🚨
- Spec Breaking changes, migration to low-code

### CODE REVIEW

File: airbyte-integrations/connectors/source-primetric/acceptance-test-config.yml [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
-      configured_catalog_path: "integration_tests/configured_catalog.json"
-      validate_schema: True
-      empty_streams: ["organization_rag_scopes"]
-      timeout_seconds: 7200
+    tests:
+      - config_path: "secrets/config.json"
+        configured_catalog_path: "integration_tests/configured_catalog.json"
+        empty_streams:
+          - name: organization_identity_providers
+            bypass_reason: Stream cannot be seeded with sandbox credentails
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-primetric/acceptance-test-config.yml:25]:
```suggestion
            bypass_reason: Stream cannot be seeded with sandbox credentials
```

Comment (askarpets) [reply airbyte-integrations/connectors/source-primetric/acceptance-test-config.yml:25]: Same for the next two

**Review by askarpets — CHANGES_REQUESTED** [review]
Good job! Please also add unit tests back where possible

**Comment by btkcodedev** [comment]
@askarpets All changes done, Updated docs, CI green

File: airbyte-integrations/connectors/source-primetric/source_primetric/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,1502 @@
...
+            rag_scope_uuid:
+              type:
+                - "null"
+                - string
+            text:
+              type:
+                - "null"
+                - string
+            value:
+              type:
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-primetric/source_primetric/manifest.yaml:1392]: This should be
```
type:
    - "null"
    - integer
```

**Review by askarpets — CHANGES_REQUESTED** [review]
@btkcodedev thanks for the updates, left one comment and please add unit tests back

File: airbyte-integrations/connectors/source-primetric/source_primetric/manifest.yaml (hunk 2 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-primetric/source_primetric/manifest.yaml:1392]: Done

**Comment by btkcodedev** [comment]
Tests fixed and CI green
Please continue merging process :bow:

**Comment by btkcodedev** [comment]
@askarpets, the credential trail period is almost over. Please review and merge it as soon as possible 🙏.

**Review by askarpets — APPROVED** [review]
@btkcodedev thanks for all updates you've made!

**Review by girarda — APPROVED** [review]

**Pull request merged**